modules/test/playwright/pages/message-boards/MessageBoardsWidgetPage.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page} from '@playwright/test';
+
+import {ApiHelpers} from '../../helpers/ApiHelpers';
+import getRandomString from '../../utils/getRandomString';
+import {performLogout} from '../../utils/performLogin';
+import {WidgetPage} from '../layout-admin-web/WidgetPage';
+export class MessageBoardsWidgetPage {
+	readonly apiHelpers: ApiHelpers;
+	readonly page: Page;
+	readonly widgetPage: WidgetPage;
+
+	constructor(page: Page) {
+		this.apiHelpers = new ApiHelpers(page);
+		this.page = page;
+		this.widgetPage = new WidgetPage(page);
+	}
+
+	async addMessageBoardsPortlet(site: Site) {
+		const layout = await this.apiHelpers.jsonWebServicesLayout.addLayout(
+			site.id,
+			getRandomString()
+		);
+
+		await this.page.goto(
+			'/web' + site.friendlyUrlPath + layout.friendlyURL
+		);
+
+		await this.widgetPage.addPortlet('Message Boards');
```

Comment:
Can we modify this to add a portlet in the main content, not the sidebar?

Reply:
I'm not sure how we could do it, since I'm adding it just clicking the Add Content button (the "+") and it is been placed in first column automatically

![Captura de pantalla 2024-06-12 a las 9 40 31](https://github.com/liferay-content-management/liferay-portal/assets/2728470/56cac480-17ac-48cd-8f97-b568dc67a621)

Maybe we could try to improve the widgetPage in further PRs if needed (in this case it doesn't seem to be relevant since tests are green)

Reply:
marked as resolved because we created a task for it